package.json

```diff
@@ -35,9 +35,7 @@
   },
   "activationEvents": [
     "onStartupFinished",
-    "onView:fileExplorer",
-    "onCommand:scope-to-this.scope",
```

Comment:
These are needed in some cases where you run the commands before VS Code is loaded or file explorer is opened. Please revert